src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -289,24 +293,39 @@ def __init__(self):
 
         return result, pool_root
 
-    def _get_pool_status(
-            self,
-            osdmap,
-            pools,
-            threshold=3.0,
-    ):
-        assert threshold >= 2.0
+    #def _get_final_pg_target_and_ratio(self, capacity_ratio, root_map, root_id, p, bias):
 
-        crush_map = osdmap.get_crush()
-
-        root_map, pool_root = self.get_subtree_resource_status(osdmap, crush_map)
+    #    even_ratio = 1 / root_map[root_id].pool_count
+    #    used_ratio = capacity_ratio
+    #    is_used = False
 
-        df = self.get('df')
-        pool_stats = dict([(p['id'], p['stats']) for p in df['pools']])
+    #    if used_ratio > even_ratio:
+    #        is_used = True
+    #        root_map[root_id].pool_count -= 1
+    #    else:
+    #        continue
+    #    final_ratio = max(used_ratio, even_ratio)
 
-        ret = []
+    #    root_map[root_id].pg_left -= (final_ratio * root_map[root_id].pg_target)
+    #    # So what proportion of pg allowance should we be using?
+    #    pool_pg_target = (final_ratio * root_map[root_id].pg_target) / p['size'] * bias
+    #    final_pg_target = max(p['options'].get('pg_num_min', PG_NUM_MIN),
+    #                              nearest_power_of_two(pool_pg_target))
+    #    
+    #    return final_pg_target, final_ratio, pool_pg_target, is_used
 
-        # iterate over all pools to determine how they should be sized
+    def _get_ret(
```

Comment:
could use a more descriptive name (maybe ```_calc_pool_targets()``` or something, and a docstring explaining the is_used parameter